src/components/procedure/_semgrep-code-display-tainted-data.mdx

```diff
@@ -10,5 +10,5 @@ To view the detailed path of tainted data with dataflow traces:
      ![Click View details if Group by Rule is enabled](/img/cloud-platform-findings-group-by-rule-view-details.png#md-width)
    - If **No grouping** view is enabled, click the **header hyperlink** on the card of the finding. In the example on the screenshot below, it is the **tainted-sql-string**.
      ![Click View details if No grouping is enabled](/img/cloud-platform-findings-no-grouping-view-details.png#md-width)
-1. In the <i class="fa-solid fa-droplet"></i> **Data flow** <i class="fa-regular fa-circle-info"></i> section, you can see the source, traces, and sink of the tainted data. The example below displays the path of tainted data across multiple files because Semgrep Pro Engine was enabled.
+1. In the <i class="fa-solid fa-droplet"></i> **Data flow** <i class="fa-regular fa-circle-info"></i> section, you can see the source, traces, and sink of the tainted data. The example below displays the path of tainted data across multiple files because Semgrep Pro Engine was enabled. Clicking on each location will highlight it in the context of the file or link directly to your source code manager (GitHub, GitLab, etc.).
```

Comment:
```suggestion
1. In the <i class="fa-solid fa-droplet"></i> **Data flow** <i class="fa-regular fa-circle-info"></i> section, you can see the source, traces, and sink of the tainted data. The example below displays the path of tainted data across multiple files because Semgrep Pro Engine was enabled. Clicking on each location highlights it in the context of the file, while clicking on the file path redirects you to your source code manager.
```